app/private_publisher.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
```

Comment:
Why is this comment in this file but not others? (Might just be a pythonism Im not aware of)